smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/formurl/FormURLEncodeCustomizable.kt

```diff
@@ -1,14 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0.
- */
-
-package software.amazon.smithy.swift.codegen.integration.serde.formurl
-
-import software.amazon.smithy.model.shapes.Shape
-
-interface FormURLEncodeCustomizable {
```

Comment:
This customization is no longer needed because this behavior is now built into the FormURL Writer.